nextflow.config

```diff
@@ -66,15 +66,21 @@ profiles {
 
   'conda' {
     conda.enabled = true
-    conda.channels = 'seqera,conda-forge,bioconda,defaults'
+    conda.channels = 'conda-forge,bioconda,defaults'
   }
-  
+
+  'mamba' {
+    conda.enabled = true
+    conda.useMicromamba = true
+    conda.channels = 'conda-forge,bioconda,defaults'
```

Comment:
worth to remove `defaults`?

Reply:
oops! Definitely, that was a mistake on my part sorry.

Reply:
https://github.com/nextflow-io/rnaseq-nf/pull/29